src/Pages/Signup/Components/Layout.scss

```
@@ -0,0 +1,31 @@
+@import '../SignUp.scss';
+
+.SignUpForm {
+  margin: 0 auto;
+  background-color: white;
+  width: 562px;
+  height: 600px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 54px 0 54px;
+  border: 1px solid rgba($color: #000000, $alpha: 0.2);
+  .SignUpTitle {
+    padding: 40px 0 40px 0;
+    font-size: 30px;
+    font-weight: bold;
+    em {
+      color: $realOrange;
+    }
+  }
+  .SignUpBtn {
```

```suggestion
  }
  
  .SignUpBtn {
```
새로운 셀렉터 앞에서는 한줄 띄워주세요.
컨벤션은 팀원과의, 리뷰어와의 약속입니다! 꼭 지켜주시길 🙏

---

네 변경하겠습니다 !! 감사합니다 !